RELEASE.md

```diff
@@ -203,16 +229,12 @@ To release a new version of Spyder you need to follow these steps:
 
 * `git tag -a vX.X.X -m "Release X.X.X"`
 
-* Update version in `__init__.py` (add 'dev0' and increment minor)
+* Update version in `__init__.py` (add 'a1', 'dev0' and increment patch version)
```

Comment:
Will `a1` be always corresponding or an instruction to update that needs to be added to?

Reply:
Nop, that's the case only for stable versions. I'll update the instruction to cover pre-releases as well.